src/Card/_variables.scss

```diff
@@ -2,24 +2,43 @@
-$card-spacer-x:                     1.25rem !default;
-$card-border-width:                 $border-width !default;
-$card-border-radius:                $border-radius !default;
-$card-border-color:                 rgba($black, .125) !default;
-$card-inner-border-radius:          subtract($card-border-radius, $card-border-width) !default;
-$card-cap-bg:                       rgba($black, .03) !default;
-$card-cap-color:                    null !default;
-$card-height:                       null !default;
-$card-color:                        null !default;
-$card-bg:                           $white !default;
-
-$card-img-overlay-padding:          1.25rem !default;
-
-$card-group-margin:                 $grid-gutter-width / 2 !default;
-$card-deck-margin:                  $card-group-margin !default;
-$card-grid-margin:                  $card-group-margin !default;
-
-$card-columns-count:                3 !default;
-$card-columns-gap:                  1.25rem !default;
-$card-columns-margin:               $card-spacer-y !default;
+$card-spacer-y:                      .75rem !default;
+$card-spacer-x:                      1.25rem !default;
+$card-border-width:                  $border-width !default;
+$card-border-radius:                 $border-radius !default;
+$card-border-color:                  rgba($black, .125) !default;
+$card-border-focus-color:            rgba($black, .5) !default;
+$card-inner-border-radius:           subtract($card-border-radius, $card-border-width) !default;
+$card-cap-bg:                        rgba($black, .03) !default;
+$card-cap-color:                     null !default;
+$card-height:                        null !default;
+$card-color:                         null !default;
+$card-bg:                            $white !default;
+
+$card-img-overlay-padding:           1.25rem !default;
+
+$card-group-margin:                  $grid-gutter-width / 2 !default;
+$card-deck-margin:                   $card-group-margin !default;
+$card-grid-margin:                   $card-group-margin !default;
+
+$card-columns-count:                 3 !default;
+$card-columns-gap:                   1.25rem !default;
+$card-columns-margin:                $card-spacer-y !default;
+
+$card-divider-bg:                    $light-400 !default;
+$card-divider-margin-y:              $card-spacer-y !default;
+
+$card-footer-actions-gap:            .5rem !default;
+
+$card-logo-left-offset:              1.5rem !default;
+$card-logo-bottom-offset:            1rem !default;
+$card-logo-left-offset-horizontal:   0.4375rem !default;
+$card-logo-bottom-offset-horizontal: 0.4375rem !default;
+
+$card-logo-width:                    7.25rem !default;
+$card-logo-height:                   4.125rem !default;
+
+$card-image-border-radius:           .3125rem !default;
+$card-logo-border-radius:            0.25rem !default;
+
+$level-1-box-shadow:                 0 .0625rem .125rem rgba(0, 0, 0, .15), 0 .0625rem .25rem rgba(0, 0, 0, .15) !default;
```

Comment:
Hmm, so the `@edx/brand-edx.org` theme specifies in its `_variables.scss` a few variables related to various elevation levels ([example](https://github.com/edx/brand-edx.org/blob/master/paragon/_variables.scss#L380-L385)).

I'm wondering if we should have something similar defined in the Paragon core `_variables.scss` file so that we are using consistent elevation levels from a theme perspective? That way, the card can use any theme overrides of the elevation values, like the brand-edx.org theme.

Reply:
Yeah, it makes sense, added new variables to Paragon core (I've taken the values from [Figma spec](https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-Design-System?node-id=9076%3A3783)).